packages/indexer-agent/src/agent.ts

```diff
@@ -990,13 +990,17 @@ export class Agent {
     targetDeployments: SubgraphDeploymentID[],
     eligibleAllocations: Allocation[],
   ): Promise<void> {
+    const logger = this.logger.child({ function: 'reconcileDeployments' })
     // ----------------------------------------------------------------------------------------
     // Ensure the network subgraph deployment is _always_ indexed
     // ----------------------------------------------------------------------------------------
-    this.multiNetworks.map(async ({ network }) => {
+    await this.multiNetworks.map(async ({ network }) => {
       if (network.networkSubgraph.deployment) {
         const networkDeploymentID = network.networkSubgraph.deployment.id
         if (!deploymentInList(targetDeployments, networkDeploymentID)) {
+          logger.trace('Ensuring Network Subgraph is indexed', {
```

Comment:
Do we really want to log this on every loop?

Reply:
I think we can include it as a boolean field in another existing log, like the child log for this function or the `Reconcile deployments` log.